ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -63,10 +61,7 @@ export default function GasTiming({
       (priority && priority !== previousMaxPriorityFeePerGas) ||
       (fee && fee !== previousMaxFeePerGas)
     ) {
-      getGasFeeTimeEstimate(
```

Comment:
Hmmm... so this PR did the opposite: https://github.com/MetaMask/metamask-extension/pull/11697/files

Reply:
That PR was created to respond to a bug

Reply:
Meaning that either `priority` and `fee` are of a different number type than they were due to some other change. Or there are some scenarios where they can be decimal GWEI, and some where they can be hex WEI

Reply:
hmmm right. Lets see

Reply:
Oh looks like maybe in swaps?

Reply:
No nevermind. Just string decimals.

Reply:
Do you remember what the issue/bug # was? Its not in the PR description

Reply:
I can't remember. We found and fixed it mid zoom call

I will see if I can recreate now